Powershell-PoC-Environment-Setup.ps1

```diff
@@ -199,15 +268,124 @@ function Invoke-AzureRmPowershellDSCIIS
                                     -AutoUpdate -Force -Verbose
 } 
 
+# NEW: 0.00.00.0001
+Function New-RandomString
+{
+ $CombinedCharArray = @()

```

Comment:
For all variables, please use camel case for variable names, e.g. $combinedCharArray instead of $CombinedCharArray.